docs/src/main/paradox/docs/releases/v1.4-to-v1.5-migration.md

```diff
@@ -0,0 +1,384 @@
+# v1.4 To v1.5 Migration
+
+The v1.5 release of Delta includes a lot of improvements like storing the compacted/expanded forms of resources to get immutability and improve performance.
+
+(To have a complete list of improvements, see here //TODO Insert link to release notes).
+
+These improvements require to replay all existing events to a new keyspace and to reindex all data to elasticsearch/Blazegraph in order to:
+
+* Adopt the new event model for every event
+* Detect and report broken resources in the system.
+
+Depending on the number and the complexity of existing resources and schemas in the system, 
+the migration may last several days (about 25,000 events per hour) to complete.
+
+Nonetheless, migration does not require to stop an interruption of service of Delta 1.4 to be performed, so a double-run of delta instances is recommended during this period.
+
+**Please do not perform any write operations as they may have an impact on the ongoing migration.**
```

Comment:
Do you mean to not perform any write operations on the 1.5.x instance while migrating, right? Write operations can be performed on the 1.4.x instance, I guess.
If that's the case I'd be explicit here, since there might be misunderstandings.

To get a good formatting of this section you can wrap that block into

```
@@@ note { .warning }

my text here

@@@
```